tekton/ci-workspace/catalog/trigger.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: catalog-pull-request
+spec:
+  interceptors:
+    - github:
+        secretRef:
+          secretName: ci-webhook
+          secretKey: secret
+        eventTypes:
+          - pull_request
+    - cel:
+        filter: >-
+          body.repository.full_name == 'catalog' &&
```

Comment:
I don't think this will match, `full_name` would be `tektoncd/catalog`. 
You use `body.repository.name == 'catalog'` instead.

Reply:
addressed :sweat_smile: